src/fonduer/parser/parser.py

```diff
@@ -637,11 +637,12 @@ def parse(self, document, text):
         stack = []
 
         root = lxml.html.fromstring(text)
-        document.text = text
 
         # flattens children of node that are in the 'flatten' list
         if self.flatten:
             lxml.etree.strip_tags(root, self.flatten)
+        # Assign the text, which was stripped of the 'flatten'-tags, to the document
+        document.text = lxml.etree.tostring(root).decode("utf-8")
```

Comment:
why should we have decode here? User might use different encoding right?

Reply:
Originally, document.text was a string. The return value of lxml.etree.tostring is a bytes object.